ceno_host/tests/test_elf.rs

```diff
@@ -105,3 +129,39 @@ fn run(state: &mut VMState) -> Result<Vec<StepRecord>> {
     eprintln!("Emulator ran for {} steps.", steps.len());
     Ok(steps)
 }
+
+#[test]
+fn test_sorting() -> Result<()> {
```

Comment:
These two examples are pulled in from https://github.com/scroll-tech/ceno/pull/753 to show that we can actively use the standard library.